samples/highcharts/demo/stellar-chart/demo.js

```diff
@@ -0,0 +1,232 @@
+const colors = ['#6CDDCA', '#C771F3', '#4D90DB', '#FAB776'];
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'bubble',
+        polar: true,
+        height: 600,
+        events: {
+            render() {
+                const chart = this,
+                    pieSeries = chart.series[1];
+                pieSeries.customLabel = fillCenter(100, '1990s-2020s', chart,
+                    pieSeries.customLabel);
+            }
+        }
+    },
+    data: {
+        csv: document.getElementById('csv').innerHTML,
+        seriesMapping: [{
+            name: 0,
+            x: 1,
+            y: 2,
+            z: 3,
+            'custom.lightYears': 4,
+            'custom.planetMass': 5,
+            'custom.stellarMagnitude': 6,
+            'custom.discoveryDate': 7
+        }]
+    },
+    title: {
+        text: 'The 240 Closest Planets to the Earth other than our solar system'
+    },
+    subtitle: {
+        text: 'Using bubble series in polar coordinate system along with pie series'
+    },
+    colorAxis: {
+        min: 0,
+        max: 12,
+        stops: [
+            [0, colors[0]],
+            [1 / 3, colors[1]],
+            [2 / 3, colors[2]],
+            [1, colors[3]]
+        ]
+    },
+    legend: {
+        enabled: false
+    },
+    pane: {
+        innerSize: '42%',
+        size: '95%',
+        background: [{
+            backgroundColor: '#f7f7f7',
+            borderWidth: 0
+        }, {
+            backgroundColor: '#fff',
+            borderWidth: 0,
+            outerRadius: '42%'
+        }]
+    },
+    xAxis: {
+        tickInterval: 1,
+        min: 0,
+        max: 30,
+        gridLineWidth: 0,
+        labels: {
+            enabled: false
+        },
+        lineWidth: 0
+    },
+    yAxis: {
+        tickInterval: 1,
+        labels: {
+            enabled: false
+        },
+        gridLineWidth: 0.5,
+        gridLineColor: '#BBBAC5',
+        gridLineDashStyle: 'longdash'
+    },
+    tooltip: {
+        borderWidth: 1,
+        backgroundColor: '#fff',
+        shadow: false,
+        outside: true,
+        hideDelay: 20,
+        formatter: function (tooltip) {
+        // Hide tooltip for pie series
+            if (this.series.options.type === 'pie') {
+                return false;
+            }
+            // If not null, use the default formatter
+            return tooltip.defaultFormatter.call(this, tooltip);
+        }
```

Comment:
This formatter can probably be omitted by just disabling tooltip for pies. `plotOptions.pie.enableMousetracking: false`.

Reply:
But then we will lose the possibility to use `mouseOver` and `mouseOut` events on pie slices to update custom text inside the circle and opacity of bubbles...